app/src/split/bluetooth/central.c

```diff
@@ -67,6 +67,13 @@ static u8_t split_central_notify_func(struct bt_conn *conn,
 		}
 	}
 
+	bt_conn_le_param_update(conn, BT_LE_CONN_PARAM(0x0006, 0x0006, 30, 400));
+
+	struct bt_conn_info info;
+
+	bt_conn_get_info(conn, &info);
+
+	LOG_DBG("Interval: %d, Latency: %d, PHY: %d", info.le.interval, info.le.latency, info.le.phy->rx_phy);
```

Comment:
This section needs to not happen on each keypress from the peripheral. Maybe we should have a check to see if the interval and latency is incorrect inside here as a fix? It may be more efficient to just update every single time though.